examples/container/memeater/Cargo.toml

```diff
@@ -4,4 +4,5 @@ version = "0.1.0"
 authors = ["ESRLabs"]
 edition = "2018"
 
-[dependencies]
\ No newline at end of file
+[dependencies]
+nix = "0.17.0"
```

Comment:
In order to keep the examples small, tiny and examplish we shouldn't add any deps here. Adding `nix` just for the page size is probably a little bit too much.